README.md

```diff
@@ -11,7 +11,8 @@ this project includes first page of portfolio website designed specifically to w
 
 ## Getting started 
```

Comment:
### Your Summary
![summary-fixable](https://user-images.githubusercontent.com/93039359/178550588-1d7eacb4-21b8-42cc-ad9b-3a6647a23c97.png)

### Summary Example
![summry-example](https://user-images.githubusercontent.com/93039359/178550602-939a964f-920f-402c-8234-78eb1640558b.png)

**[Optional]**

- [ ] Your summary is descriptive but it would be nice if you make your summary shown in the example above :blush: then your pull request will look professional :+1: